### ISSUE TITLE
The height of menu is calculated incorrectly with extra css

### ISSUE THREAD

**Issue by kylephp** [snapappointments/bootstrap-select#2229]
Hi @caseyjhol,
I want to add some custom styling on `<li>` (padding + border) but I found the height of inner menu will be calculated incorrectly and the scroll therefore will not be working properly.
JSfiddle: https://jsfiddle.net/mugncLhe/1/
When you use arrow down key to go down, the option 9  + 10 is not visible and the scroll is not updated.
Any idea how can I do it in the proper way?
TIA

**Comment by caseyjhol** [comment]
I'd recommend not applying any styling on the `li` element directly - although that should probably be taken into account when calculating size in bootstrap-select.

Using

```css
.dropdown-menu>li>a {
    padding: 5px 20px;
    border-bottom: 1px dotted #ccc;
}
```
should give you better results.

https://jsfiddle.net/caseyjhol/s1nLuz0x/

**Comment by kylephp** [comment]
Thanks @caseyjhol 
It does the trick! Are u planning to fix it with a better way in next release?

**Comment by caseyjhol** [comment]
Yes, styling on the `li` element will be taken into account in the next release.

**Comment by kylephp** [comment]
sounds good! (y)

**Comment by caseyjhol** [comment]
Released in [v1.13.8](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.8)!

**Comment by msbtterswrth** [comment]
I have no idea if I should add a new ticket or comment here.

While you fixed the issue with li css, the same issue appears with css on the live-search div, or .bs-searchbox. 

In my case, I'm adding margin bottom to the searchbox which is not accounted for in the js height, so the last item gets cut off a wee bit. Here is an example - https://dev.muhealth.org/doctors; type in cancer. You'll see the last item 'vulvar cancer' is slightly cut off. 

If I remove the margin from the search box, all is well! Can you update this fix to account for the live-search box too, please?
